app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -250,24 +281,40 @@ class AddDeadlineActivity : AppCompatActivity() {
         alertDialogBuilder.show()
     }
 
+    /**
+     * Setup an AlertDialog that will select the group of the deadline
+     */
+    fun selectGroups(view: View) {
+        val alertDialogBuilder = AlertDialog.Builder(this)
+
+        // Set the title of the Dialog
+        alertDialogBuilder.setTitle("Select Group:")
```

Comment:
Why this comment? The method is self-explanatory! Comments of this kind just make the file longer and make more lines to change if you have to modify something, and are completely useless in my opinion.